lib/redisFactory.js

```diff
@@ -47,4 +47,6 @@ exports.task = (opt) => {
       },
     },
   });
+  queue.setMaxListeners(queue.getMaxListeners() + 1);
```

Comment:
I got a warning for memory leak cause too many job TTL listeners added, so I use `setMaxListeners` to prevent it.
here is an related ticket, https://github.com/Automattic/kue/issues/822

Reply:
got it, not sure if +1 is enouth though.